Swift_Alert/Swift_Alert/AuthenticationView/AuthenticationViewModel.swift

```diff
@@ -8,51 +8,76 @@ final class AuthenticationViewModel: ObservableObject {
     // MARK: - Constants
 
     private enum Constants {
-        static var detailViewTagText = "DetailView"
-        static var emptyText = ""
-        static var minPasswordNumber = 6
-        static var maxPasswordNumber = 15
-        static var phoneNumber = 11
+        static let detailViewTagText = "DetailView"
+        static let emptyText = ""
+        static let minPasswordNumber = 6
+        static let maxPasswordNumber = 15
+        static let phoneNumber = 18
+        static let progressMaxCount = 10
+        static let timeInterval = 0.3
+        static let zeroNumber = 0
+        static let oneNumber = 1
+        static let startScale: CGFloat = 0
     }
 
     // MARK: - Public Properties
 
     @Published var lastPasswordText = Constants.emptyText
```

Comment:
```suggestion
    @Published var lastPasswordText = 
```
я же писал в важном что так больше не делать